sway/input/seat.c

```diff
@@ -438,6 +437,7 @@ static void seat_apply_input_config(struct sway_seat *seat,
 
 static void seat_configure_pointer(struct sway_seat *seat,
 		struct sway_seat_device *sway_device) {
+	seat_configure_xcursor(seat);
```

Comment:
Does this have any undesirable side-effects if there are multiple pointers on the seat?

Reply:
The laptop I am testing it on has both a "nub" pointer device and a trackpad. No side effects, both of them work fine.